website/docs/guides/resource-tagging.html.md

```diff
@@ -0,0 +1,173 @@
+
+<!-- TOC depthFrom:2 -->
+
+- [Getting Started with Resource Tags](#getting-started-with-resource-tags)
+- [Ignoring Changes to Specific Tags](#ignoring-changes-to-specific-tags)
+  - [Ignoring Changes in Individual Resources](#ignoring-changes-in-individual-resources)
+  - [Ignoring Changes in All Resources](#ignoring-changes-in-all-resources)
+- [Managing Individual Resource Tags](#managing-individual-resource-tags)
+
+<!-- /TOC -->
+
+## Getting Started with Resource Tags
+
+Terraform AWS Provider resources that support resource tags implement a consistent argument named `tags` which accepts a key-value map, e.g.
+
+```hcl
+resource "aws_vpc" "example" {
+  # ... other configuration ...
+
+  tags = {
+    Name = "MyVPC"
+  }
+}
+```
+
+The tags for the resource are wholly managed by Terraform. The only exception to this is tag keys beginning with `aws:` as these are managed by AWS services and usually cannot be edited or deleted. Any non-AWS tags added to the VPC outside of Terraform will be proposed for removal on the next Terraform execution. Missing tags or those with incorrect values from the Terraform configuration will be proposed for addition or update on the next Terraform execution. Advanced patterns that can adjust these behaviors for special use cases, such as Terraform AWS Provider configurations that affects all resources and the ability to manage resource tags for resources not managed by Terraform, can be found later in this guide.
+
+For most environments and use cases, this is the typical implementation pattern, whether it be in a standalone Terraform configuration or within a [Terraform Module](/docs/modules/). The Terraform configuration language also enables less repetitive configurations via [variables](/docs/configuration/variables.html), [locals](/docs/configuration/locals.html), or potentially a combination of these, e.g.
+
+```hcl
+# Terraform 0.12 and later syntax
+variable "additional_tags" {
+  default     = {}
+  description = "Additional resource tags"
+  type        = map(string)
+}
+
+resource "aws_vpc" "example" {
+  # ... other configuration ...
+
+  # This configuration combines some "default" tags with optionally provided additional tags
+  tags = merge(
+    var.additional_tags,
+    {
+      Name = "MyVPC"
+    },
+  )
+}
+```
+
+## Ignoring Changes to Specific Tags
+
+Systems outside of Terraform may automatically interact with the tagging associated with AWS resources. These external systems may be for administrative purposes, such as a Configuration Management Database, or the tagging may be required functionality for those systems, such as Kubernetes. This section shows methods for prevent Terraform from showing differences for specific tags.
```

Comment:
`for prevent Terraform` -> `for preventing Terraform`

Reply:
Thanks! Fixed.